ansible/libvirt_create_vms.yml

```diff
@@ -3,7 +3,7 @@
   include_role:
     name: stackhpc.libvirt-vm
   vars:
-    libvirt_vm_default_console_log_dir: "{{ console_log_directory }}"
+    libvirt_vm_default_console_log_dir: "{{ log_directory }}"
```

Comment:
This is /var/log/tenks/ rather than the previous value of /var/log/tenks/console_logs, is that intended?

Reply:
Yes. I originally thought about having `console_logs` and `vbmc_logs` subdirectories, but VBMC complains about missing log handlers when logging to a file in a subdirectory (e.g. `/var/log/tenks/vbmc_logs`), so I thought it would work fine to just put them all in `/var/log/tenks` and name the VBMC log files differently.